src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -21,142 +21,153 @@ signupBanner:
   text: Monitor and improve your entire stack. 100GB free. Forever.
 ---
 
-Browser uses a [JavaScript snippet](/docs/browser/new-relic-browser/page-load-timing/instrumentation-page-load-timing), also referred to as an "agent," to instrument your app's webpages. The JavaScript collects data for browser monitoring. To install the browser agent, you can choose from a number of deployment options.
+import browserInstallStep1 from 'images/browser_screenshot-full_install-step-1.png'
 
-If you don't have one already, create a New Relic account. It's free, forever.
+import browserInstallStep2 from 'images/browser_screenshot-full_install-step-2.png'
 
-<InlineSignup/>
+import browserInstallStep3 from 'images/browser_screenshot-full_install-step-3.png'
 
+import browserInstallStep4 from 'images/browser_screenshot-full_install-step-4.png'
 
-## Enable browser monitoring [#enable]
+import browserInstallStep5 from 'images/browser_screenshot-full_install-step-5.png'
 
-Browser Pro+SPA is the default agent when you enable browser monitoring. This automatically gives you access to all of our browser monitoring features. For more information about the browser monitoring options, see [Browser agent types](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#agent-types) in this document.
+To get started with browser monitoring, you'll need to add a JavaScript snippet, which we call our browser agent, to your webpage HTML. Then the agent will collect browser data like JavaScript errors, core web vitals, and other performance data, and will sent that data to New Relic where you can view it in your New Relic UI.  
 
-To enable browser monitoring:
+Now, let's get started setting up browser monitoring in the steps below. 
 
-1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, click **Add data**, then click **Browser and mobile**.
-2. Follow the instructions in the UI to select which data source for browser monitoring you want to use for your app, such as **New Relic Browser**.
-3. Generate some traffic for your app, then wait a few minutes for data to appear in New Relic.
-4. Optional: After installation is complete and you are seeing data, go to the [**App settings** page](/docs/browser/new-relic-browser/configuration/browser-settings-ui-options-apdex-geography/) for additional agent configuration, or to change the [browser agent type](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#agent-types).
+## Before you begin 
 
+1. Review our browser compatibility and requirements for:
```

Comment:
Do I have to? :) What's the harm if we have a step 1 where they don't review the requirements?

I'd rather stick this in an optional section or something.